sdk/cosmos/azure-cosmos/azure/cosmos/_utils.py

```diff
@@ -69,3 +69,17 @@ def get_index_metrics_info(delimited_string: Optional[str]) -> Dict[str, Any]:
         return result
     except (json.JSONDecodeError, ValueError):
         return {}
+
+
+def is_base64_encoded(data: str) -> bool:
+    if data is None:
+        return False
+    try:
+        base64.b64decode(data, validate=True).decode('utf-8')
```

Comment:
Currently it will be called twice per request. A typical continuation token will look like the following (for single feed range coverage):

![image](https://github.com/user-attachments/assets/445a3d10-dcf4-47c2-afae-a0c16c232ea8)

For V1 continuation token, it can be converted to long, so changed to use int(continuation) for validating the continuation format

Reply:
changed to validate whether the continuation token is number format